src/module.c

```diff
@@ -11142,6 +11068,105 @@ int moduleUnload(sds name) {
     return C_OK;
 }
 
+int moduleConfigsWereSet(RedisModule *module) {
```

Comment:
please add a top comment describing what the function does and what it's used for.